cvcl/views/assignment.py

```diff
@@ -155,14 +155,20 @@ def post(self, request, **kwargs):
             if vmd.timezone:
                 data_dict['timezone'] = vmd.timezone
 
-            if vmd.timezone:
+            if vmd.hostname:
                 data_dict['hostname'] = vmd.hostname
 
             user_data = '#cloud-config\n' + yaml.dump(data_dict, default_flow_style=False)
 
             if vmd.powershell_script:
                 user_data = '#ps1_sysnative\n' + vmd.powershell_script
 
+            if vmd.script_per_boot:
```

Comment:
The link your provided. Isn't it reading from a file? I am a bit confused. Are we having a file field in the form or are we taking in the user_data scripts that are pasted into the fields and encoding them into MIME format.